src/main.cpp

```diff
@@ -186,100 +208,192 @@ void addPackOptions(po::options_description& desc)
-        ("name,n", po::value<string>(), "Model name.")
-        ("version,V", po::value<string>(), "Model version.")
-        ("description,d", po::value<string>(), "Model description.")
-        ("labels,l", po::value<string>(), "Labels file name.")
-        ("date-time", po::value<string>(), "Date/time the model was created (optional). Date/time is normally set "
+        ("type,t", po::value<string>()->value_name("TYPE"),
+            "Type of the input model. Currently supported types:\n \t - caffe")
+        ("json,j", po::value<string>()->value_name("PATH"),
+            "Model metadata in JSON format. Command line parameters will override "
+            "entries in this file if present.")
+        ("name,n", po::value<string>()->value_name("NAME"), "Model name.")
+        ("category,C", po::value<string>()->value_name("CATEGORY"), "Model category.")
+        ("version,V", po::value<string>()->value_name("VERSION"), "Model version.")
+        ("description,d", po::value<string>()->value_name("DESCRIPTION"), "Model description.")
+        ("labels,l", po::value<string>()->value_name("PATH"), "Labels file name.")
+        ("label-names", po::value<vector<string>>()->value_name("LABEL1 [LABEL2 ...]"),
+            "A list of label names.")
+        ("date-time", po::value<string>()->value_name("DATE_TIME"),
+            "Date/time the model was created (optional). Date/time is normally set "
             "automatically, specifying this argument is not recommended. Must be in ISO format, which is "
             "YYYYMMDDTHHMMSS,fffffffff where T is the date-time separator. i.e. 20020131T100001,123456789")
-        ("model-size,w", po::cvSize_value(), "Classifier model size. Model parameters will "
+        ("model-size,w", po::cvSize_value()->value_name("WIDTH [HEIGHT]"), "Classifier model size. Model parameters will "
             "override this if present. Must specify width and height. i.e. --model-size 128 128")
-        ("bounding-box,b", po::cvRect2d_value(), "Training area bounding box. Must specify four "
+        ("bounding-box,b", po::cvRect2d_value()->value_name("W S E N"),
+            "Training area bounding box (optional). Must specify four "
             "coordinates: west longitude, south latitude, east longitude, and north latitude. i.e. "
-            "--bounding-box -84.06163 37.22197 -84.038803 37.240162")
-        ("image-type,i", po::value<string>(), "Image type. i.e. jpg")
-        ("color-mode,c", po::value<string>(), "Color mode. Model parameters will override this if present. Must be one of the following: grayscale, rgb, multiband")
+            "--bounding-box -180 -90 180 90")
+        ("color-mode,c", po::value<string>()->value_name("MODE"),
+            "Color mode. Model parameters will override this if present. Must be one of the following: grayscale, rgb, multiband")
+        ("resolution,r", po::cvSize2d_value()->value_name("WIDTH [HEIGHT]"),
```

Comment:
"Second argument is missing" is a tall order in general, but the "required argument is missing" message comes from `boost::program_options`, I don't set it. I'll see what I can do to make this work better.